README.md

```diff
@@ -64,7 +64,15 @@ module.exports = {
 
 ### Loading Scoped Styles
 
-You need to specify scoped flag and loaders for style files such as `style-loader` and `css-loader`. Note that they must be `enforce: post` to inject scope id into styles before they are processed by them.
+For an explanation of scoped styles, see the [vue-loader docs](https://vue-loader.vuejs.org/en/features/scoped-css.html). 
+
+Html and style files need to be imported using the loader syntax: `require("./file.html?style=./file.css")`. You also need modify your webpack config as follows:
```

Comment:
Yup. You're absolutely right. Sorry I didn't follow your lead on that. Two points though:
1. Isn't `import withRender from './app.html?style=./app.css'`  a default import? As least with webpack and typescript, for this to work (I think?) the loader needs to export using the syntax `exports.default = ...` rather than  `module.exports = ...` ? Maybe babel works differently?
2. It looks like there's an annoying limitation with the typescript `declare module '*.html'` syntax where you can't use two wildcards like `declare module '*.html?style=*'`. This essentially forced me to use the require() syntax to get typescript to compile. Maybe I'm missing something here though.